app/controllers/api/v1/conversations_controller.rb

```diff
@@ -14,40 +14,42 @@ class ConversationsController < Api::V1::BaseController
       end
 
       rescue_from ActiveRecord::RecordNotFound do
-        render(
-          json:   {error: I18n.t("conversations.not_found")},
-          status: :not_found
-        )
+        render json: I18n.t("api.endpoint_errors.conversations.not_found"), status: :not_found
```

Comment:
Please keep line length < 80 chars

Reply:
Did that flag on the style check for line length?  It didn't on my machine.  I'll correct.

Reply:
The limit on diaspora's pronto is 120.

Reply:
@Flaburgan are we using 120 or 80 then?  If it's 120 I am inclined to leave this as is.

Reply:
I don't like 120. I prefer 80. It's better for collaboration.